### DISCUSSION TITLE
On code style

### DISCUSSION THREAD

**Discussion by i-make-robots** [MarginallyClever/Makelangelo-software#427] [General]
Some people have trouble with this speaker because of his political views.  Ad hominem aside, he makes excellent points about clean code.

https://www.youtube.com/watch?v=7EmboKQH8lM

**Comment by PPAC37** [comment]
Very interesting thank you.
Too much staging.
But points that tell me that I still have a lot to learn to make good code.

**Comment by i-make-robots** [comment]
staging?

**Reply by PPAC37** [reply]
For me, in the 1h48 of video there must be 30 minutes that can be skipped .. that's what I probably call wrongly staging.

**Reply by i-make-robots** [reply]
Life is short and time is precious.  On the other hand, he makes convincing arguments that some people need to hear.
